transport/src/test/java/io/netty5/channel/DefaultChannelOutboundBufferTest.java

```diff
@@ -80,232 +66,10 @@ public void flushingEmptyBuffers() throws Exception {
-                assertEquals(expected, buffers[i]);
-            }
-            return true;
-        });
-
-        release(buffer);
-        buf.close();
-    }
-
-    @Test
-    public void testNioBuffersExpand2ByteBuf() {
-        TestChannel channel = new TestChannel();
-
-        ChannelOutboundBuffer buffer = new ChannelOutboundBuffer(channel);
-
-        CompositeByteBuf comp = compositeBuffer(256);
-        ByteBuf buf = directBuffer().writeBytes("buf1".getBytes(CharsetUtil.US_ASCII));
-        for (int i = 0; i < 65; i++) {
-            comp.addComponent(true, buf.copy());
-        }
-        buffer.addMessage(comp, comp.readableBytes(), channel.newPromise());
-
-        assertEquals(0, buffer.nioBufferCount(), "Should still be 0 as not flushed yet");
-        buffer.addFlush();
-        ByteBuffer[] buffers = buffer.nioBuffers();
-        assertEquals(65, buffer.nioBufferCount());
-        for (int i = 0;  i < buffer.nioBufferCount(); i++) {
-            if (i < 65) {
-                assertEquals(buffers[i], buf.internalNioBuffer(buf.readerIndex(), buf.readableBytes()));
-            } else {
-                assertNull(buffers[i]);
-            }
-        }
-        release(buffer);
-        buf.release();
-    }
-
-    @Test
-    public void testNioBuffersExpand2() {
-        TestChannel channel = new TestChannel();
-
-        ChannelOutboundBuffer buffer = new ChannelOutboundBuffer(channel);
-
-        Buffer buf = BufferAllocator.offHeapUnpooled().copyOf("buf1", CharsetUtil.US_ASCII);
-        @SuppressWarnings("unchecked")
-        var sends = Stream.generate(() -> buf.copy().send()).limit(65).collect(Collectors.toList());
-        CompositeBuffer comp = BufferAllocator.offHeapUnpooled().compose(sends);
-
-        buffer.addMessage(comp, comp.readableBytes(), channel.newPromise());
-
-        assertEquals(0, buffer.nioBufferCount(), "Should still be 0 as not flushed yet");
-        buffer.addFlush();
-        ByteBuffer[] buffers = buffer.nioBuffers();
-        assertEquals(65, buffer.nioBufferCount());
-        assertEquals(1, buf.countReadableComponents());
-        buf.forEachReadable(0, (index, component) -> {
-            assertEquals(0, index);
-            ByteBuffer expected = component.readableBuffer();
-            for (int i = 0;  i < buffer.nioBufferCount(); i++) {
-                if (i < 65) {
-                    assertEquals(expected, buffers[i]);
-                } else {
-                    assertNull(buffers[i]);
-                }
-            }
-            return true;
-        });
-
-        release(buffer);
-        buf.close();
-    }
-
-    @Test
-    public void testNioBuffersMaxCountByteBuf() {
-        TestChannel channel = new TestChannel();
-
-        ChannelOutboundBuffer buffer = new ChannelOutboundBuffer(channel);
-
-        CompositeByteBuf comp = compositeBuffer(256);
-        ByteBuf buf = directBuffer().writeBytes("buf1".getBytes(CharsetUtil.US_ASCII));
```

Comment:
The new implementation only supports `nioBuffers(...)` for `Buffer` which is good enough as we want to get rid of `ByteBuf` anyway.